.github/workflows/check.yml

```diff
@@ -4,18 +4,20 @@
 on:
   pull_request:
   push:
-    branches:
-      - main
```

Comment:
Temporarily removed this so I can run GHA on my fork under my `patch-1` branch. If you approve my PR workflow I can revert this.

Reply:
Can you please revert the change, you can create a PR to your own `main` branch to check the GHA

Reply:
Sounds good. I've sent in a new one here: https://github.com/ably/ably-php/pull/167